### PR TITLE
PMM-11730 alert section

### PR DESCRIPTION
**Pull request by filipmikes1**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
https://jira.percona.com/browse/PMM-11730

Fixes #

**Special notes for your reviewer**:
image

### CODE REVIEW

File: public/app/percona/check/components/CheckInfoAlert/CheckInfoAlert.styles.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+
+export const getStyles = ({ v1: { colors: v1Colors } }: GrafanaTheme2) => ({
```
Comment (matejkubinec) [inline public/app/percona/check/components/CheckInfoAlert/CheckInfoAlert.styles.ts:5]: You can access the blue link color from `theme.colors.text.link` in `GrafanaTheme2`.

image

Don't know if that's the same as the one you need, so feel free to ignore this if it isn't.

File: public/app/percona/check/components/CheckInfoAlert/CheckInfoAlert.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import { getStyles } from './CheckInfoAlert.styles';
+
+export const ChecksInfoAlert = () => {
+  const { result } = useSelector(getPerconaSettings);
+  const { isConnectedToPortal } = result!;
+  const styles = useStyles2(getStyles);
+
+  return (
+    <>
+      {!isConnectedToPortal && (
```
Comment (matejkubinec) [inline public/app/percona/check/components/CheckInfoAlert/CheckInfoAlert.tsx:19]: You could do an early return here.

```ts
if (isConnectedToPortal) {
   return null;
}

return (<AlertLocalStorage ... />);
```

This way you would avoid the fragment and coditional in component render.

File: public/app/percona/shared/components/Elements/AlertLocalStorage/AlertLocalStorage.constants.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export const DAY = 1000 * 60 * 60 * 24;
```
Comment (matejkubinec) [inline public/app/percona/shared/components/Elements/AlertLocalStorage/AlertLocalStorage.constants.ts:1]: `DAY_IN_MS` might be more descriptive.

File: public/app/percona/shared/components/Elements/AlertLocalStorage/AlertLocalStorage.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  }, [uniqueName]);
+
+  const handleCloseAlert = () => {
+    setToLocalStorage(uniqueName, true, 7);
+    setShowAlert(false);
+  };
+
+  return (
+    <>
+      {showAlert && (
```
Comment (matejkubinec) [inline public/app/percona/shared/components/Elements/AlertLocalStorage/AlertLocalStorage.tsx:65]: The same as with `CheckInfoAlert`.

File: public/app/percona/check/components/CheckInfoAlert/CheckInfoAlert.styles.ts (hunk 1 above)
Comment (filipmikes1) [reply public/app/percona/check/components/CheckInfoAlert/CheckInfoAlert.styles.ts]: Yes it is same, changed it.

File: public/app/percona/check/components/CheckInfoAlert/CheckInfoAlert.tsx (hunk 2 above)
Comment (filipmikes1) [reply public/app/percona/check/components/CheckInfoAlert/CheckInfoAlert.tsx]: Good tip :)

**Review by matejkubinec — APPROVED** [review]

**Review by matejkubinec — APPROVED** [review]

**Review by fabio-silva — APPROVED** [review]

**Pull request merged**